src/Goteo/Util/Form/Type/TypeaheadType.php

```diff
@@ -25,13 +25,15 @@ public function configureOptions(OptionsResolver $resolver)
     {
         parent::configureOptions($resolver);
         $resolver->setDefaults([
+            'entry_type' => TextType::class,
+            'allow_add' => true,
```

Comment:
If we don't change the base type from TextType to CollectionType, maybe we don't need this two fields, right?

Reply:
Good spot! I've removed both of them here, and actually I was able to remove the `entry_type` from the `MultipleTypeaheadType` class too